plugins/modules/cloud/scaleway/scaleway_function_namespace.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Scaleway Serverless function namespace management module
+#
+# Copyright (c) 2022, Guillaume MARTINEZ <lunik@tiwabbit.fr>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_function_namespace
+short_description: Scaleway Function namespace management
+version_added: 5.8.0
```

Comment:
```suggestion
version_added: 6.0.0
```
The next release which accepts new plugins/modules will be 6.0.0

Reply:
Fixed in : 6effe81e6d1be87392e9544fd193a17706acc135